dev/CommonStyles/TextBlock_themeresources.xaml

```diff
@@ -0,0 +1,54 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <Style x:Key="BaseTextBlockStyle" TargetType="TextBlock">
```

Comment:
In the testapp, did you see if the change in DefaultTextBoxStyle kept? For example, the CornerRadius.
Less likely, we will need things like:
```
<Style x:Key="BaseTextBlockStyle" TargetType="TextBlock" BasedOn="{StaticResource DefaultTextBoxStyle }>
```

Reply:
Are you sure that we want to derive from the DefaultTextBoxStyle for the BaseTextBlockStyle? Aren't those targeting two different controls?

Reply:
You are right. I didn't see the box and block difference